src/request/import/ImportFile.js

```diff
@@ -161,6 +170,7 @@ class ImportFile {
         const result = [{
             keyId: key.id,
             keyType: key.type,
+            ...(this._label ? { keyLabel: this._label } : {}),
```

Comment:
question: could this be simplified?

```js
keyLabel: this_label ? this_label : null, 
// or null-coalescing if allowed
// or || would probably be fine also
```

Reply:
The alternative would be returning an `undefined`. I think omitting the key instead is nicer.